src/Models/Services/Utils.ts

```diff
@@ -811,12 +811,13 @@ export const getMockTimeSeriesDataArrayInLocalTime = (
 ): Array<Array<TimeSeriesData>> => {
     const toInMillis = Date.now();
     const fromInMillis = toInMillis - agoInMillis;
-    return Array.from({ length: lengthOfSeries }).map(() =>
-        Array.from({ length: numberOfDataPoints }, () => ({
+    return Array.from({ length: lengthOfSeries }).map(() => {
+        const maxLimitVariance = Math.floor(Math.random() * 500); // pick a max value between 0-500 as this timeseries value range to add more variance for values of different timeseries in independent y axes
```

Comment:
uhhhhhh why do we use random here??? That seems like we are making it non-deterministic? 🤔🤔🤔

Reply:
this is mock data and I need to generate a random value